examples/django_app/requirements.txt

```diff
@@ -1,2 +1,4 @@
 django>=1.8.0,<2.0.0
 chatterbot>=0.5.0
+gunicorn
+whitenoise
```

Comment:
I think adding these steps to the Django documentation sounds like a good solution, it allow the example app to remain simple while still providing recommendations to developers who are looking to create a hosted solution on a server.